### PR TITLE
PIN-4948: Agreement process in production

### PR DESCRIPTION
**Pull request by galales**

### CODE REVIEW

File: kubernetes/overlays/api-gateway/prod/configmap.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ metadata:
 data:
   ACCEPTED_AUDIENCES: "{{M2M_JWT_AUDIENCE}}"
 
-  AGREEMENT_PROCESS_URL: "http://{{AGREEMENT_PROCESS_SERVICE_NAME}}:{{BACKEND_SERVICE_PORT}}/{{AGREEMENT_PROCESS_APPLICATION_PATH}}/{{AGREEMENT_PROCESS_INTERFACE_VERSION}}"
+  AGREEMENT_PROCESS_URL: "http://{{AGREEMENT_PROCESS_SERVICE_NAME}}:{{BACKEND_SERVICE_PORT}}"
```
Comment (beetlecrunch) [inline kubernetes/overlays/api-gateway/prod/configmap.yaml:9]: 😍

File: kubernetes/overlays/tenant-process/prod/configmap.yaml [hunk 2]
```diff
@@ -5,4 +5,4 @@ metadata:
   namespace: {{NAMESPACE}}
 data:
   ACCEPTED_AUDIENCES: "{{UI_JWT_AUDIENCE}},{{M2M_JWT_AUDIENCE}},{{INTERNAL_JWT_AUDIENCE}}"
-  AGREEMENT_PROCESS_URL: "http://{{AGREEMENT_PROCESS_SERVICE_NAME}}:{{BACKEND_SERVICE_PORT}}/{{AGREEMENT_PROCESS_APPLICATION_PATH}}/{{AGREEMENT_PROCESS_INTERFACE_VERSION}}"
+  AGREEMENT_PROCESS_URL: "http://{{AGREEMENT_PROCESS_SERVICE_NAME}}:{{BACKEND_SERVICE_PORT}}"
```
Comment (beetlecrunch) [inline kubernetes/overlays/tenant-process/prod/configmap.yaml:8]: 😍

File: kubernetes/overlays/tenant-process/qa/configmap.yaml [hunk 3]
```diff
@@ -5,4 +5,4 @@ metadata:
   namespace: {{NAMESPACE}}
 data:
   ACCEPTED_AUDIENCES: "{{UI_JWT_AUDIENCE}},{{M2M_JWT_AUDIENCE}},{{INTERNAL_JWT_AUDIENCE}}"
-  AGREEMENT_PROCESS_URL: "http://{{AGREEMENT_PROCESS_SERVICE_NAME}}:{{BACKEND_SERVICE_PORT}}/{{AGREEMENT_PROCESS_APPLICATION_PATH}}/{{AGREEMENT_PROCESS_INTERFACE_VERSION}}"
+  AGREEMENT_PROCESS_URL: "http://{{AGREEMENT_PROCESS_SERVICE_NAME}}:{{BACKEND_SERVICE_PORT}}"
```
Comment (beetlecrunch) [inline kubernetes/overlays/tenant-process/qa/configmap.yaml:8]: 😍

File: kubernetes/configs/prod/backend_for_frontend.sh [hunk 4]
```diff
@@ -21,4 +21,11 @@ BFF_RATE_LIMITER_TIMEOUT="300.milliseconds"
 
 SUPPORT_SAML_AUDIENCE="selfcare.interop.pagopa.it"
 SUPPORT_SAML_CALLBACK_URL="https://selfcare.interop.pagopa.it/ui/it/assistenza/scelta-ente"
-SUPPORT_SAML_CALLBACK_ERROR_URL="https://selfcare.interop.pagopa.it/ui/it/assistenza/errore"
\ No newline at end of file
+SUPPORT_SAML_CALLBACK_ERROR_URL="https://selfcare.interop.pagopa.it/ui/it/assistenza/errore"
+
+IMPORT_ESERVICE_CONTAINER="interop-application-import-export-test"
```
Comment (beetlecrunch) [inline kubernetes/configs/prod/backend_for_frontend.sh]:
```suggestion
IMPORT_ESERVICE_CONTAINER="interop-application-import-export-prod"
```

File: kubernetes/configs/prod/backend_for_frontend.sh [hunk 5]
```diff
@@ -21,4 +21,11 @@ BFF_RATE_LIMITER_TIMEOUT="300.milliseconds"
...
+IMPORT_ESERVICE_PATH="{{NAMESPACE}}/eservices-import"
+EXPORT_ESERVICE_CONTAINER="interop-application-import-export-test"
```
Comment (beetlecrunch) [inline kubernetes/configs/prod/backend_for_frontend.sh]:
```suggestion
EXPORT_ESERVICE_CONTAINER="interop-application-import-export-prod"
```

**Review by beetlecrunch — DISMISSED** [review]

File: kubernetes/configs/prod/backend_for_frontend.sh (hunk 4 above)
Comment (galales) [reply kubernetes/configs/prod/backend_for_frontend.sh]: thanks!

**Review by beetlecrunch — APPROVED** [review]

**Review by micdes-pagopa — APPROVED** [review]

**Pull request merged**